doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc

```diff
@@ -10,61 +10,86 @@ You can use the process instance migration (PIM) service to create, export and e
-          auth-method: BASIC
-          security-domain: pim
-        security-constraints:
-          - url-pattern: /*
-            roles: [ admin ]
-          - url-pattern: /health/*
-  datasources:
-    data-sources:
-      pimDS:
-        driver-name: h2
-        connection-url: jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
-        user-name: DS_USERNAME
-        password: DS_PASSWORD
+quarkus:
+  class-loading:
+    removed-artifacts: com.oracle.database.jdbc:ojdbc8,com.ibm.db2:jcc,com.microsoft.sqlserver:mssql-jdbc
+  package:
+    type: mutable-jar
+    user-providers-directory: providers
+  http:
+    auth:
+      basic: true
+      policy:
+        main-policy:
+          roles-allowed: admin
+      permission:
+        main:
+          paths: /*
+          policy: main-policy
+        public:
+          paths: /q/health/*
+          policy: permit
+          methods: GET
   security:
-    security-domains:
-      pim:
-        classic-authentication:
-          login-modules:
-            UsersRoles:
-              code: UsersRoles
-              flag: required
-              module-options:
-                usersProperties: application-users.properties
-                rolesProperties: application-roles.properties
-kieservers:
-  - host: http://localhost:8080/kie-server/services/rest/server
-    username: KIESERVER_USERNAME
-    password: KIESERVER_PASSWORD
-  - host: http://localhost:8280/kie-server/services/rest/server
-    username: KIESERVER_USERNAME
-    password: KIESERVER_PASSWORD1
+    users:
+      file:
+        realm-name: pim_file
+        enabled: true
+        plain-text: true
+        users: users.properties
+        roles: roles.properties
+    jdbc:
+      realm-name: pim_jdbc
+      enabled: true
+      principal-query:
+        sql: SELECT u.password, u.role FROM users u WHERE u.username=?
+    ldap:
+      realm-name: pim_ldap
+      enabled: true
+      dir-context:
+        url: ldap://override-when-needed
+      identity-mapping:
+        search-base-dn: ou=users,o=YourCompany,c=ES
+# Quartz configuration
+  quartz:
+    store-type: ram
+    start-mode: forced
+  resteasy:
+    path: /rest
+  datasource:
+    db-kind: h2
+    jdbc:
+      url: jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    username: sa
+    password: sa
+  hibernate-orm:
+    database:
+      generation: update
+pim:
+  auth-method: file
+"%int":
+  kieservers:
+  - host: http://localhost:18080/kie-server/services/rest/server
+    username: kieserver
+    password: kieserver1!
 ----
 . Start the PIM service:
 +
 [source,subs="attributes+"]
 ----
 $ java -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
 ----
-. To enable auto-detection of a JDBC driver by Thorntail, add the `JAR` file of the JDBC driver name to the `thorntail.classpath` system property. For example:
+. To re-augment a Quarkus configuration you need to build your application with `quarkus.package.type=mutable-jar`:
```

Comment:
I'd add the reasons why someone would need to re-agument the build. The main reason would be to add more dependencies or to change build time properties as it is the example using the mariadb datasource
You may refer to the Quarkus documentation if needed (https://quarkus.io/guides/reaugmentation)

Reply:
Added the reasoning to the step and added a link the the relevant guide.